kubewatch.yaml

```diff
@@ -16,17 +16,18 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 4730af84d096fabf3d0c5a2aae15d666baac141b
+      expected-commit: 957dc4a61943136220e2c32a15881928eec81fb4
       repository: https://github.com/robusta-dev/kubewatch
       tag: v${{package.version}}
 
-  - uses: go/bump
-    with:
-      deps: google.golang.org/protobuf@v1.33.0 golang.org/x/net@v0.23.0
-
+  # Temporary Fix: Upstream go.sum file does not accurately capture package dependencies.
   - runs: |
-      make build
-      install -Dm755 kubewatch "${{targets.destdir}}"/usr/bin/kubewatch
+      go mod tidy
```

Comment:
we don't need a step to run go mod tidy. go/build does it.

Reply:
why don't we use `make build` ?

Reply:
I thought so too, but I tried not including that block: https://github.com/wolfi-dev/os/pull/21756/commits/ba7a0878617e671a6fee5c04a1768ad2d386578d, and the tests don't pass. As per https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/build.yaml, I don't think go mod tidy is run unless there's a dependency to bump (as there was before the package update).

As to why we don't use `make build`, @ajayk suggested we should migrate to `go/build` whenever possible